src/openscad.cc

```diff
@@ -920,6 +934,8 @@ int main(int argc, char **argv)
   init_mimalloc();
 #endif
 
+  assertValidPath();
```

Comment:
Not sure I understand, is your goal to assert (fail with an error), or silently find the closest valid path?

Reply:
Yeah, assert is a bat verb, I was thinking of silently finding the closes path